stable/raccoon/README.md

```diff
@@ -0,0 +1,97 @@
+# Raccoon
+
+A [helm](https://helm.sh/) chart for [Raccoon](https://github.com/odpf/raccoon)
+
```

Comment:
Please clarify what this chart deploys or includes in it's reference. Basically installs Raccoon as a service, telegraf as sc, cm etc.,